scripts/poa-bytecode.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const solc = require('solc');
+const Web3 = require('web3');
+const web3 = new Web3(new Web3.providers.HttpProvider("https://core.poa.network"));
+
+main();
+
+async function main() {
+	console.log('PoaNetworkConsensus compilation...');
+	const compiled = solc.compile({
+		sources: {
+			'': fs.readFileSync('../contracts/PoaNetworkConsensus.sol').toString()
+		}
+	}, 1, function (path) {
+		return {contents: fs.readFileSync('../contracts/' + path).toString()}
+	});
+	const abi = JSON.parse(compiled.contracts[':PoaNetworkConsensus'].interface);
+	let bytecode = compiled.contracts[':PoaNetworkConsensus'].bytecode;
+	
+	const contract = new web3.eth.Contract(abi);
+	const deploy = await contract.deploy({data: '0x' + bytecode, arguments: [process.env.MASTER_OF_CEREMONY, []]});
```

Comment:
we don't need to deploy POANetworkConsensus smart-contract somewhere to know its final bytecode. Please, take a look https://github.com/poanetwork/poa-test-setup/blob/master/prepareMoCNode.js#L69-L70, where final bytecode is generated.

Reply:
@vbaranov, the function `deploy` doesn't deploy a contract. It just prepares the bytecode to deploy.

Reply:
@varasev you are right. I kept in mind `deployer.deploy` from Truffle, which sends transaction. Nevertheless, `contract.deploy` is more expensive, then `web3.eth.abi.encodeParameters`: 7.935ms vs 6.056ms

Reply:
@vbaranov it is not time-critical operation, but ok, next time I will use web3.eth.abi.encodeParameters. Thanks.